recipes/spyder-vim/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "spyder-vim" %}
```

Comment:
```suggestion
{% set name = "Spyder-Vim" %}
```

Proper-case

Reply:
We generally don't capitalize the names for the conda-forge package.

Reply:
Right, and indeed it is under `package.name` below, and in most other uses. The intent here was to use the canonical display case in the Jinja `name` variable here, so it could be lowercased when used programmatically (with the `lower` filter) and left intact when used as a display name. However, I don't see it actually used as a display name anywhere currently, and the PyPI name is also lowercase, so indeed this isn't really necessary.

Reply:
I will skip this suggestion and leave the name as it is following the PyPI package name